teachertool/src/App.tsx

```diff
@@ -35,28 +38,37 @@ export const App = () => {
                 await loadValidatorPlansAsync();
                 await tryLoadLastActiveChecklistAsync();
 
-                // Test notification
-                showToast({
-                    ...makeToast("success", "🎓", 2000),
-                    hideIcon: true,
-                    hideDismissBtn: true,
-                    className: "app-large-toast",
-                });
-
                 setInited(true);
                 logDebug("App initialized");
             });
         }
     }, [ready, inited]);
 
-    return !inited ? (
+    useEffect(() => {
+        async function checkAuth() {
```

Comment:
```suggestion
        async function checkAuthAsync() {
```

Reply:
Oh, did you want to keep this without the `async` since you have the `async` one in the auth client?

Reply:
Either way is fine, as far as I'm concerned. It's only defined in the context of this useEffect anyway. I'll update it for consistency.